Source code/Embedded/vendor/cros-chameleon/edid/basic_display.py

```diff
@@ -0,0 +1,334 @@
+# Copyright 2014 The Chromium OS Authors. All rights reserved.
```

Comment:
Huh.. Chromium doesn't follow PEP 8. Interesting.

Reply:
yeah I thought so too. I didn't diverge much from their original style to ease upstreaming if they wanted it. However, given that the json to edid script was broken by typos and has been for years, I suspect they no longer use this. My fork is here: https://github.com/rpavlik/cros-chameleon

Shame, because this is a really awesomely-useful bit of software.  (Somewhere I had added more features for Sensics - to support either raw binary extensions or DisplayID extensions, don't remember which - 
maybe 2 years ago but I can't find that code...)